editor/code_editor.cpp

```diff
@@ -1851,6 +1854,7 @@ void CodeTextEditor::_apply_settings_change() {
 	}
 
 	text_editor->set_code_hint_draw_below(EDITOR_GET("text_editor/completion/put_callhint_tooltip_below_current_line"));
+	text_editor->set_middle_mouse_paste_enabled(EDITOR_GET("text_editor/behavior/input/middle_mouse_button_paste"));
```

Comment:
Shouldn't be a need to set it here as well, `CodeTextEditor::update_editor_settings` should be enough, it gets called when the settings change as well.

Reply:
Done!